src/MarginTrading.Backend/Application.cs

```diff
@@ -93,11 +96,16 @@ public sealed class Application
                         HandleStpOrderbook,
                         _rabbitMqService.GetMsgPackDeserializer<ExternalExchangeOrderbookMessage>());
                 }
+
+                var registry = new Registry();
+                registry.Schedule<OvernightSwapJob>().ToRunEvery(0).Days().At(_marginSettings.OvernightSwapCalculationHour, 0);
+                JobManager.Initialize(registry);
+                JobManager.JobException += info => _logger.WriteError(ServiceName, nameof(JobManager), info.Exception);
             }
             catch (Exception ex)
             {
                 _consoleWriter.WriteLine($"{ServiceName} error: {ex.Message}");
-                await _logger.WriteErrorAsync(ServiceName, nameof(StartApplicationAsync), null, ex);
+                await _logger.WriteErrorAsync(ServiceName, "Application.RunAsync", null, ex);
```

Comment:
Why not nameof? )

Reply:
Must be cherry-picked from Nuno's commit, so can't tell why.